README.rst

```diff
@@ -24,6 +24,22 @@ Parsing:
    12345
    ['Marxus, Karlus', 'Lindgren, Astrid']
 
+A file path can also be used to read RIS files. If an encoding is not specified in ``load``, the default system encoding will be used.
+
+.. code:: python
+
+   >>> import os
+   >>> from pathlib import Path
+   >>> from pprint import pprint
+   >>> import rispy
+   >>> filepath = 'tests/data/example_utf_chars.ris'
+   >>> p = Path(filepath)
+   >>> entries = rispy.load(p, path_encoding='utf-8')
```

Comment:
change parameter name to `encoding`? to be consistent with python open()  - https://docs.python.org/3/howto/unicode.html#reading-and-writing-unicode-data

Reply:
I thought about it earlier, the reason I chose `path_encoding` was because I wanted to specify that it was only useful when you supplied a path. it's probably still better to leave it as `encoding`, though, and just document it